packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -246,6 +257,10 @@ export default {
          */
         async initialise () {
             this.setAuthToken(this.authToken);
+            if (!this.isLoggedIn) {
```

Comment:
Ultra nit: add an empty line before this one, please.

Reply:
Fixed